service/internal/class.tx_mksearch_service_internal_Index.php

```diff
@@ -722,6 +801,24 @@ protected function getSecondsToKeepQueueEntries()
         return $secondsToKeepQueueEntries ? $secondsToKeepQueueEntries : $thirtyDaysInSeconds;
     }
 
+    /**
+     * Returns the amount of minutes to keep being indexed entries in queue
+     * before resetting them.
+     *
+     * @return int
+     */
+    protected function getMinutesToKeepBeingIndexedEntries()
+    {
+        $minutesToKeepBeingIndexedEntries = tx_rnbase_configurations::getExtensionCfgValue(
+            'mksearch',
+            'minutesToKeepBeingIndexedEntries'
+        );
+
+        return $minutesToKeepBeingIndexedEntries ? $minutesToKeepBeingIndexedEntries : 60;
```

Comment:
Can be shortened to "$minutesToKeepBeingIndexedEntries ?: 60;"

Reply:
done